plugins/flutter_aepedge/test/flutter_aepedge_test.dart

```diff
@@ -44,4 +46,148 @@ void main() {
       expect(await Edge.extensionVersion, testVersion);
     });
   });
+
+  group('sendEvent', () {
+    final Map<dynamic, dynamic> xdmData = {"eventType": "SampleEventType"};
+    final Map<String, dynamic> data = {"free": "form", "data": "example"};
+
+    final Map<String, dynamic> experienceeventData = {
+      "xdmData": xdmData,
+      "data": data,
+      "datasetIdentifier": "datasetExample",
+    };
+
+    //setup experienceEvent
+    final ExperienceEvent experienceEvent =
+        ExperienceEvent(experienceeventData);
+
+    final String eventHandleType = "state:store";
+    final List<dynamic> eventHandlePayload = [
+      {"maxAge": 1800, "value": "testValue1", "key": "keyExample1"},
+      {"maxAge": 34128000, "value": "testValue2", "key": "keyExample2"}
+    ];
+
+    final Map<dynamic, dynamic> eventHandleExpectedData = {
+      "type": eventHandleType,
+      "payload": eventHandlePayload
+    };
+
+    //setup returned and expected EventHandles
+    final EventHandle expectedEvent = EventHandle(eventHandleExpectedData);
+    final EventHandle returnedEvent = EventHandle(eventHandleExpectedData);
+
+    final List<EventHandle> expectedResponse = [expectedEvent];
+    final List<dynamic> returnedResponse = [returnedEvent.data];
```

Comment:
can we use only expectedEvent, expectedEvent and setMockMethod to return expectedResponse instead of duplicating the expected* & returned*?

Reply:
I need to return List<dynamic> returnedResponse for setMockMethod.  It doesn't take List<EventHandle>.